app/views/foods/shopping_list.html.erb

```diff
@@ -0,0 +1,29 @@
+
+<div class='c-flex recipe-details'>
+  <h2 class="center_self">Shopping List</h2>
+  <div class='shop_list flex'>
+    <p>Amount of food items to buy:<span><%= @qty %></span></p>
+    <p>Total value of item needed:<span> $ <%= @sum %></span></p>
+  </div>
+   <table class='food_table'>
+      <tr>
+         <th>Food</th>
+         <th>Quantity</th>
+         <th>Unit Price</th>
+
+      </tr>
+
+      <% @foods.each do |food| %>
+         <%if current_user == food.user%>
+          <tr>
+                <td> <%= food.name %> </td>
+                <td> <%= food.quantity %>   <%= food.measurement_unit  %></td>
+                <td> $ <%= food.price * food.quantity.to_i %> </td>  
+           </tr>
+
+         <%end%>
+      <% end %>
```

Comment:
- [ ] The general shopping list should show a list of food that is missing for all recipes of the logged-in user. I can see that you are simply displaying all the food in the database which is not accurate, unfortunately. Please make this change 👍

Reply:
Hi @codepantha ,
Thank you very much for your effort. The correction will be made in my next commit. :+1: